src/mon/MonCommands.h

```diff
@@ -175,6 +175,12 @@ COMMAND("auth get-or-create " \
 	"name=caps,type=CephString,n=N,req=false", \
 	"add auth info for <entity> from input file, or random key if no input given, and/or any caps specified in the command", \
 	"auth", "rwx", "cli,rest")
+COMMAND("auth fs " \
```

Comment:
I'd like that syntax better, too. Is there precedent for implementing commands in the "wrong" monitor like this?